frame/bags-list/src/lib.rs

```diff
@@ -155,7 +153,7 @@ pub mod pallet {
 
 	/// How many ids are registered.
 	// NOTE: This is merely a counter for `ListNodes`. It should someday be replaced by the
-	// `CountedMaop` storage.
+	// `CountedMap` storage.
 	#[pallet::storage]
```

Comment:
sidenote: counted map is now merged in master

Reply:
yeah I have a self-note to move this and the staking counters to this new type. Neither should require any migration.